cmd/start/main.go

```diff
@@ -432,3 +437,38 @@ func getCommitSHA(dir string) (string, error) {
 	}
 	return strings.TrimSpace(string(content)), nil
 }
+
+func gitLfsInUse(logger logging.LeveledLoggerInterface, dir string) bool {
+	stdout, stderr, err := command.RunInDir("git", []string{
+		"lfs",
+		"ls-files",
+		"--all",
+	}, dir)
+	if err != nil {
+		logger.Errorf(string(stderr))
+		log.Fatal(err)
+	}
+
+	// no error, so count lines/files
+	files := 0
+	endOfLine := '\n'
+	for _, c := range string(stdout) {
+		if c == endOfLine {
+			files++
+		}
+	}
```

Comment:
I think you can simplify this:
```
return strings.Split(strings.TrimRight(string(stdout), "\n"), "\n") > 0
```

Reply:
That said, do you need to split into lines at all? Or would something simple like `return  strings.TrimSpace(string(stdout)) != ""` achieve the same?